src/makielayout/defaultattributes.jl

```diff
@@ -11,8 +11,30 @@ function lift_parent_attribute(::Nothing, attr::Symbol, default_value)
 end
 
 
+function wong_colors(alpha)
+    colors = [
+        RGB(0/255, 114/255, 178/255), # blue
+        RGB(230/255, 159/255, 0/255), # orange
+        RGB(0/255, 158/255, 115/255), # green
+        RGB(213/255, 94/255, 0/255), # vermillion
+        RGB(204/255, 121/255, 167/255), # reddish purple
+        RGB(86/255, 180/255, 233/255), # sky blue
+        RGB(240/255, 228/255, 66/255), # yellow
+    ]
+    @. RGBAf0(red(colors), green(colors), blue(colors), alpha)
+end
+
 function default_attributes(::Type{Axis}, scene)
     attrs, docdict, defaultdict = @documented_attributes begin
+        "Cycler"
+        cycler = Cycler2()
+        "palettes"
+        palettes = Attributes(
+            color = wong_colors(1),
+            patchcolor = wong_colors(0.6),
+            linestyle = [:solid, :dash, :dot, :dashdot, :dashdotdot],
+            marker = [:circle, :rect, :utriangle, :diamond, :xcross, :dtriangle, :pentagon, :star8]
```

Comment:
also fine with me, although I also think we should equalize these markers better. they seem to be differently sized to me, which is probably because of the underlying glyphs.